superset/views/base_api.py

```diff
@@ -195,21 +195,24 @@ class BaseSupersetModelRestApi(ModelRestApi):
         }
     """
 
-    related_field_filters: Dict[str, Union[RelatedFieldFilter, str]] = {}
+    base_related_field_filters: Dict[str, BaseFilter] = {}
     """
-    Declare the filters for related fields::
+    This is used to specify a base filter for related fields
+    when they are accessed through the '/related/<column_name>' endpoint.
+    When combined with the `related_field_filters` attribute,
+    this filter will be applied in addition to the latest::
```

Comment:
Should we move this comment to `related_field_filters` and say there that the filter is applied in addition (or on top, whichever sounds better) to `base_related_field_filters`?